src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (x) {
+      calculator.result += x;
+    },
+    subtract: function (x) {
```

Comment:
[CODE KNOWLEDGE] - The `subtract` method should also use the shorthand syntax without the `function` keyword.